mythril/laser/ethereum/svm.py

```diff
@@ -36,7 +36,7 @@ class SVMError(Exception):
 class LaserEVM:
     """The LASER EVM.
 
-    Just as Mithril had to be mined at great efforts to provide the
+    Just as Mythril had to be mined at great efforts to provide the
```

Comment:
Actually, the ore is named Mithril https://en.wikipedia.org/wiki/Mithril

Reply:
Oh, I see. I have reverted this one.